eth_portal/bridge/run.py

```diff
@@ -120,3 +124,29 @@ def load_private_keys():
             sys.exit(INVALID_KEY_ENV_ERROR)
         else:
             return keys
+
+
+def load_provider(provider_arg):
+    if provider_arg == "custom":
+        try:
+            client_id = os.environ["AUTH_CLIENT_ID"]
+            client_secret = os.environ["AUTH_CLIENT_SECRET"]
+            client_url = os.environ["AUTH_CLIENT_URL"]
+        except KeyError:
+            sys.exit(INVALID_AUTH_PROVIDER_ENV_ERROR)
+        headers = {
+            "headers": {
+                "Content-Type": "application/json",
+                "CF-Access-Client-Id": client_id,
+                "CF-Access-Client-Secret": client_secret,
+            }
+        }
+        w3 = Web3(Web3.HTTPProvider(client_url, request_kwargs=headers))
+        if not w3.isConnected():
+            raise RuntimeError(
+                "Invalid authenticated provider credentials, provider is not connected."
+            )
+    else:
+        from web3.auto.infura import w3
+
+        return w3
```

Comment:
Two things:
1. this looks like it's indented incorrectly (or a return is missing before the `else`)
2. it's surprising you didn't notice this in local manual testing, which I'm assuming you did. Can you double-check your local test to see why it didn't show up for you?

Reply:
Ahh good catch. I was testing this locally, but I think some last-minute refactoring might have been the culprit here. It's all working locally with the latest commit.